controllers/cloud.redhat.com/helpers/clowdenvs.go

```diff
@@ -100,12 +76,5 @@ func VerifyClowdEnvReady(env clowder.ClowdEnvironment) (bool, error) {
 		}
 	}
 
-	if !reconciliationSuccessful {
-		return false, errors.New("reconciliation not successful")
-	}
-	if !deploymentsReady {
-		return false, errors.New("deployments not ready")
-	}
-
-	return true, nil
+	return (reconciliationSuccessful && deploymentsReady), nil
```

Comment:
Kudos for code reduction, but why don't we need to handle these conditions anymore?

Reply:
The errors in that were being printed while waiting on the ClowdEnvironment to actually be ready. From how I understand it, if any part of the ClowdEnvironment actually fails, it would error out elsewhere so I figured I could remove the errors here since this is simply a check for if it is ready. If the ClowdEnvironment never actually comes up, that return will always output false.